assets/textures/crystal_normal.png.import

```diff
@@ -0,0 +1,35 @@
+[remap]
+
+importer="texture"
+type="CompressedTexture2D"
+uid="uid://ppco0jrr2v0q"
+path.s3tc="res://.godot/imported/crystal_normal.png-bcc547839c255a50d32bc95b49febcba.s3tc.ctex"
+metadata={
+"imported_formats": ["s3tc_bptc"],
+"vram_texture": true
+}
+
+[deps]
+
+source_file="res://assets/textures/crystal_normal.png"
+dest_files=["res://.godot/imported/crystal_normal.png-bcc547839c255a50d32bc95b49febcba.s3tc.ctex"]
+
+[params]
+
+compress/mode=2
+compress/high_quality=false
+compress/lossy_quality=0.7
+compress/hdr_compression=1
+compress/normal_map=0
```

Comment:
Is this imported with normal map settings?

Reply:
I don't think I checked that. I can re-import.
 - It has made changes. I'll update it.